src-app/biz-modules/ai-chat/components/agent-edit-modal.tsx

```diff
@@ -176,7 +176,9 @@ export const AgentEditModal = (props: Props) => {
 								onChange={(avatar: string) => setPayload({ ...payload, avatar })}
 							/>
 							<div className="ml-xs-6 mt-xs-6">
-								<img src={payload.avatar} alt="" className="h-sm-8 w-sm-8 rounded-full" />
+								{Boolean(payload.avatar) && (
```

Comment:
`""` evaluates to `false`, so no need to be too explicit, by using `Boolean()`

Reply:
I get eslint warning if it's not boolean. Or should I completely disable that rule?